grpcutil/creds.go

```diff
@@ -1,20 +1,40 @@
 package grpcutil
 
 import (
+	"crypto/tls"
+
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
+var tlsVersions = map[string]uint16{
+	"1.0": tls.VersionTLS10,
+	"1.1": tls.VersionTLS11,
```

Comment:
should we even allow anyone to choose 1.0 or 1.1? I doubt we have any use case for it (I'm not sure if we even have any need to support anything below 1.3?)